python/mqt/syrec/syrec_editor.py

```
@@ -1148,11 +1271,24 @@ def setup_actions(self) -> None:
 
+        if destringified_circuit_view_qubit_label is None:
+            show_error_dialog(
+                "Error during parsing of clicked qubit label of circuit view",
+                "Failed to convert the clicked qubit label in the circuit view: \n"
+                + stringified_circuit_view_qubit_label
+                + "\n"
+                + "into its internal DTO representation! This should not happen and indicates an internal error!",
+            )
+        else:
+            self.qubits_information_lookup.search_and_display_information_for_qubit(
+                destringified_circuit_view_qubit_label, True
```

Ruff (0.14.0)

Boolean positional value in function call

(FBT003)